static/fontscale.js

```diff
@@ -28,26 +28,28 @@ define(function (require) {
     var _ = require('underscore');
     var EventEmitter = require('events');
 
+    function FontSizeDropdown(elem, interval, scale, isFontStr) {
```

Comment:
Made function because I c&p'ed some of it from settings.js. Maybe ok to inline it